protos/feast/core/CoreService.proto

```diff
@@ -202,6 +269,19 @@ message ListStoresResponse {
     repeated feast.core.Store store = 1;
 }
 
+message ApplyEntityRequest {
+    // If project is unspecified, will default to 'default' project.
+    // If project specified does not exist, the project would be automatically created.
+    feast.core.Entity entity = 1;
```

Comment:
Which parts of the entity can be updated?

Reply:
That can be found [here](https://github.com/feast-dev/feast/pull/1014/files#diff-431a7153babcc8eca8290e9578cd7996R82).

Reply:
Does it make more sense to expose only `EntitySpecV2` instead to users, as the other (meta) fields are set by Feast?

Reply:
agreed, `EntitySpecV2` should be here instead

Reply:
Updated.